service/controller/resource/clusterconfigmap/types.go

```diff
@@ -1,5 +1,13 @@
 package clusterconfigmap
 
+type AppOperatorProvider struct {
+	Kind string `json:"kind"`
```

Comment:
We will not provider `provider.kind` to `cluster-apps-operator` and it will fetch the info from infra CRs. However, it will pass the info to the app-operator of clusters as a static value. Right?

Reply:
That is correct